cloudbuild.yaml

```diff
@@ -46,6 +46,20 @@ steps:
     args: [ 'run', 'deploy', 'sds', '--image', 'europe-west2-docker.pkg.dev/${_PROJECT_ID}/sds/sds:$SHORT_SHA',
             '--region', 'europe-west2', '--allow-unauthenticated', '--ingress', 'internal-and-cloud-load-balancing' ]
 
+  - name: 'gcr.io/cloud-builders/gcloud'
```

Comment:
Is it possible to move this step before "Run image" so that image with severe vulnerability will not be deployed at all to minimise risk?